components/cdc/src/delegate.rs

```diff
@@ -436,7 +416,7 @@ impl Delegate {
             } = cmd;
             if !response.get_header().has_error() {
                 if !request.has_admin_request() {
-                    self.sink_data(index, request.requests.into());
+                    self.sink_data(index, request.requests.into(), old_value_cb.clone())?;
```

Comment:
Why not clone old value?

Reply:
To reduce costs, clone a hash map is not a light action.

Reply:
I think it may cause extra IO.

Reply:
What about wrapping the hashmap in `Rc`?

Reply:
`old_value_cb` was already wrapped in `Rc` so I think the value it captured should also be accessed in `Rc` way.